### PR TITLE
[Admin] Removes heretics from rotation

### PR DESCRIPTION
**Pull request by alexkar598**
Let me preface this with: No, I haven't died due to heretics, this is not an ided.

Heretics is a gamemode where its core mechanic is killing people to gain points and using those points, you get abilities that mainly go into two categories:
- Damage
- Healing

Both of those very much encourage combat and the way you get them(by sacrificing dead people) makes killing people, in a semi permanant way mandatory.
Don't forget about the end ability of the heretics either, which is become a very high DPS enemy that is hard to kill AND on top of that, can still use items(except for flesh path I believe) which means they can stall the shuttle for a long amount of times.

Heretics can be fun in the right context but it doesn't fit yogs where murderbone is frowned upon unlike TG where crew is allowed to help fight back antags and antags are allowed to murderbone.

tl;dr; heretics encourages murderbone via game mechanics, good fit for tg not for yogs

:cl:  alexkar598
rscdel: Heretics are no longer in rotation.
/:cl:

### CODE REVIEW

**Comment by Jarod1200** [comment]
Sombody who died to a heretic bribed him to make this

**Comment by UselessTheremin** [comment]
thanks alexkar this is a cool pr, sure it may be a bit more murderboney and more of a tg antag but people find it fun. nukeops is murderbone and people love fighting nukies, same with wizard. not a fan of this change, especially since youre admitting you either
a. dont have experience with heretics so have no clue what youre talking about
b. just got killed by a heretic

**Comment by MegaEmpirical** [comment]
... He says at the very start he wasn't killed and I see no admission that they don't know what they're talking about.

**Comment by alexkar598** [comment]
The difference between nukies and heretics is everyone has a weapon and is allowed to validhunt the hell out of the nukies.

Heretics operates under yogs rules but with a tg game design.

**Comment by LazennG** [comment]
ided pr shut up

**Comment by boodaliboo** [comment]
why is this even being considered almost all of the playerbase loves it

**Comment by alexkar598** [comment]
Bold statement to make with no proper data.

**Comment by boodaliboo** [comment]
> Bold statement to make with no proper data.

my evidence is the 1 vote up for this pr and 17 down

**Comment by MegaEmpirical** [comment]
I've seen you close PRs for having FAR less down votes too.

**Comment by Szyszkrzyneczka** [comment]
Just make heretics valid, don't remove them from the pool,

The less antag variety we have the worse the experience will be

**Comment by UselessTheremin** [comment]
fetish content

**Comment by maxion12345** [comment]
is Alexkar going orange mode

**Comment by Marmio64** [comment]
we should remove changelings next because their objectives cater towards them killing random people in a semi-permanent manner

**Comment by boodaliboo** [comment]
why has this not been closed you have closed my prs for 5 downvotes

**Comment by N3D6** [comment]
why is this not closed

**Comment by boodaliboo** [comment]
> why is this not closed

because alexkar doesnt want to even though it makes absolutely no sense to keep this pr open at this point

**Comment by LoliconSlayer** [comment]
Thumbs up so I can mark myself for death

**Comment by TheLinuxOS** [comment]
In case alexkar still feels there isn’t enough “proper data” to support that our play base likes the game mode if direct him to the recent quick fire vote
